docs/Attack_Narrative_Template.md

```diff
@@ -0,0 +1,43 @@
+# Attack Narrative - The Template
+The main goal of this document is to describe how a malicious user could exploit a vulnerability, intentionally installed on `your app's name` from secDevLabs, to `add here the vulnerability`.
+
+If you don't know [secDevLabs] or this `intended vulnerable web application` (be sure to leave a link here!) yet, you should check them before reading this narrative.
+
+---
+## 👀
+
+This first part should be about how the vulnerability was found, you should explain all the steps needed for someone to be able to replicate this vulnerability.
+
+Usually, some nice steps to include are:
+* What's the app main page? How can you access it?
+* How did you discover the vulnerability? If you used the command line, be sure to include the command used! You can include the command by doing this:
+```sh
+    ```sh
+    $ My awesome command here
+
+    ```
+```
+
+Add as many images as you can! A picture is worth more than a thousand words!
```

Comment:
Suggestion: Add at the end of this sentence: `Some examples:"

Reply:
Thanks @rafaveira3 ! I'll add this sentence above the images!